### DISCUSSION TITLE
Disabling Filepond allowpaste property

### DISCUSSION THREAD

**Discussion by byru** [filamentphp/filament#600] [Help]
Thank you for your works on this package.
I have a Rich text and two file upload components in one form. Whenever I copy paste contents that has images, the filepond automatically detected the images. Looking at filepond configuration there is a property that can be turned to false (allowPaste). How do I manage to achieve that ?

**Comment by danharrin** [comment]
Are you using the admin panel or the form builder?

Happy to accept a PR to add that property to Filepond, it must be a widespread issue.

**Reply by byru** [reply]
I am using the admin panel, sorry I didn't mention it earlier

**Comment by ManojKiranA** [comment]
Is there any method in `v3` that allows dynamically to enable and disable the `allowPaste` property
